src/App/utils/text.js

```diff
@@ -34,31 +34,31 @@ export const unpublishedTitleText = 'Unpublished'
 
 export const publishedTitleText = 'Published'
 
-export const noInProgressLessonsDescriptionText = 'You have no lessons in progress. Go claim a new lesson!'
+export const noInProgressLessonsDescriptionText = 'You\'ve got no lessons in progress.'
 
 export const noInstructorsDescriptionText = 'No instructors to show'
 
 export const noPublishedLessonsDescriptionText = 'Looks like you don\'t have any published lessons. Go complete the Get Published checklist!'
 
 export const statsTitleText = 'Stats'
 
-export const currentMonthRevenueTitleText = 'Next paycheck (estimated)'
+export const currentMonthRevenueTitleText = 'Estimated royalties'
 
 export const totalRevenueTitleText = 'Last twelve months'
 
 export const revenueLabelText = 'dollars'
 
-export const subscriberMinutesLabelText = 'subscriber minutes'
+export const subscriberMinutesLabelText = 'minutes watched'
```

Comment:
I feel this one is confusing as to me "minutes watched" means all minutes watched, which is not the case? I ask because I've had a few instructors reach out asking about this - it is only subscriber minutes.